rpc/opencdm_xdr_xdr.c

```diff
@@ -49,7 +49,10 @@ bool_t
 xdr_rpc_request_create_session (XDR *xdrs, rpc_request_create_session *objp)
 {
 	register int32_t *buf;
-
+#if WPE
+        if (!xdr_int32_t (xdrs, &objp->license_type))
+                return FALSE;
+#endif
 	 if (!xdr_array (xdrs, (char **)&objp->init_data_type.init_data_type_val, (u_int *) &objp->init_data_type.init_data_type_len, ~0,
```

Comment:
General comment about all the commits, they need proper descriptions about why the change is being made. Also consider using git -rebase to squash any typos and fix earlier commits.

The end goal for this work should be to have a single cdmiservice binary which can work with both Chromium and WPE browsers.

Reply:
Ok, maybe the github GUI way I used is limiting me from fix/feature based commits (instead to file based) & rebasing. I will use git command-line then to have cleaner commits.